pkgs/by-name/ot/otb/package.nix

```diff
@@ -0,0 +1,125 @@
+{
+  cmake,
+  callPackage,
+  fetchFromGitHub,
+  makeWrapper,
+  lib,
+  stdenv,
+  swig,
+  which,
+  boost,
+  curl,
+  gdal,
+  libsvm,
+  libgeotiff,
+  muparser,
+  muparserx,
+  opencv,
+  perl,
+  python3,
+  shark,
+  tinyxml,
+  enableFeatureExtraction ? true,
+  enableHyperspectral ? true,
+  enableLearning ? true,
+  enableMiscellaneous ? true,
+  enableOpenMP ? false,
+  enablePython ? true,
+  extraPythonPackages ? ps: with ps; [ ],
+  enableRemote ? true,
+  enableSAR ? true,
+  enableSegmentation ? true,
+  enableStereoProcessing ? true,
+}:
+assert enablePython -> python3 != null;
+let
+  inherit (lib) optionalString optionals optional;
+  pythonInputs =
+    optionals enablePython (with python3.pkgs; [ numpy ]) ++ (extraPythonPackages python3.pkgs);
+
+  otb-itk = callPackage ./itk_4_13/package.nix { };
+  otb-shark = shark.override { enableOpenMP = enableOpenMP; };
```

Comment:
Why not enable openmp by default and drop the option?

Reply:
Thanks @SuperSandro2000 !

By default it is `OFF` [c.f](https://gitlab.orfeo-toolbox.org/orfeotoolbox/otb/-/blob/develop/CMakeLists.txt?ref_type=heads#L182) so I have kept it as it is.